libraries/AP_TECS/AP_TECS.h

```diff
@@ -419,6 +419,9 @@ class AP_TECS {
     // need to reset on next loop
     bool _need_reset;
 
+    // Checks if we reset at the beginning of takeoff.
+    bool _reset_after_takeoff{false};
```

Comment:
The variable name doesn't match the function. Suggest this is renamed something like _takeoff_entry_reset_cmd

Reply:
I meant for this to be a flag to be queried and checked.
If it's OK with you, I'll rename it to `_flag_have_reset_after_takeoff`.
If not, I'll take your suggestion and call it `_takeoff_entry_reset_cmd`.

Reply:
@Georacer it also should not be initialised to false - the init just wastes flash space

Reply:
Renamed and removed initialization.